daemon/response.go

```diff
@@ -150,7 +151,7 @@ func ErrorResponse(status int) ErrorResponseFunc {
 		Status: status,
 	}
 
-	return r.SetError
+	return r.setError
```

Comment:
Ah. It used to be a `Response`. Can you suggest a different name? Before, `NotFound` was a `Response` (an `ErrorResponseFunc` was a `Response`), and `NotFound(err, "blah")` was a `Response` also; now only the latter is true.